BangazonAPI/Models/Order.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonAPI.Models
+{
+    public class Order
+    {
+        public int Id { get; set; }
+        public int CustomerId { get; set; }
+        public int PaymentTypeId { get; set; }
```

Comment:
Orders should have a list of `Products` and a list of `Customers` based on the requirements in ticket 4

Reply:
added